httpserver/service_node.cpp

```diff
@@ -320,14 +320,15 @@ void ServiceNode::bootstrap_data() {
     for (auto seed_node : seed_nodes) {
         lokid_client_.make_lokid_request(
             seed_node.first, seed_node.second, "get_n_service_nodes", params,
-            [this, seed_node, req_counter, node_count = seed_nodes.size()](const sn_response_t&& res) {
+            [this, seed_node, req_counter,
+             node_count = seed_nodes.size()](const sn_response_t&& res) {
                 if (res.error_code == SNodeError::NO_ERROR) {
                     try {
                         const block_update_t bu = parse_swarm_update(res.body);
                         on_bootstrap_update(bu);
                     } catch (const std::exception& e) {
                         LOKI_LOG(
-                            error,
+                            critical,
```

Comment:
I think it might make more sense to make these bootstrap logs warn, and the log below saying `Could not contact any of the seed nodes to get target height` to an error?

Reply:
My reasoning was that all exceptions should be handled by us and not printed to the user. If it is printed, then it must be something bad. I think we should keep the bottom one as `warn` since it might not be the node's fault that seed nodes are inaccessible.

Reply:
Updated back to error